### DISCUSSION TITLE
Does 1.20.1 Neoforged have a Forge 47.1.3 equivalent?

### DISCUSSION THREAD

**Discussion by Handrail9** [Sinytra/Connector#1287] [Q&A]
Asking because I'd like to use connector on 1.20.1 but I'd also like to use Neoforged on 1.20.1. I know it will somewhat work on the newest version of neoforged for 1.20.1 as I've tested it a bit, but I know it's unsupported and not recommended. If there isn't a version of Neoforged that is recommended I'll just have to choose between having connector or having neoforged.

**Comment by WitherTM** [comment]
any version of neoforge is equivalent to forge 47.1.3. also, the version is supported, just can't have any breaking changes (changes that break mods) to stay compatible with LexForge

**Reply by Handrail9** [reply]
Hm, alright. I was only asking because I was trying to get Bedrockify working but it causes a crash on Neoforge. I assumed it wasn't supported due to [this issue](https://github.com/Sinytra/Connector/issues/858#issuecomment-1962940482). Should I open a new issue or would it be better for me to leave Bedrockify out?

**Reply by AleksQuantum** [reply]
I also get a crash here. Says "Requested GL version 4.6 got version 4.6".
